hadoop-connector-examples/build.gradle

```diff
@@ -17,7 +17,7 @@ apply plugin: "distribution"
 
 sourceCompatibility = 1.8
 archivesBaseName = 'pravega-hadoop-examples'
-version = connectorVersion
+version = hadoopConnectorVersion
```

Comment:
It should be samples version. we can remove this variable assignment. I have opened a PR to address the hadoop connector version mismatch issue (https://github.com/pravega/pravega-samples/pull/106).

Reply:
Ok, thanks.